services/facility_queue/facility_queue.py

```diff
@@ -237,29 +257,39 @@ def service(queue):
                                 continue
 
                             lightcurve = row['lightcurve']
-                            dataurl = f"{ZTF_FORCED_URL}/{lightcurve}"
-                            try:
-                                commit_photometry(
-                                    dataurl,
-                                    altdata,
-                                    req.id,
-                                    instrument.id,
-                                    followup_request.requester.id,
-                                    session,
-                                )
-                                req.status = 'complete'
-                                session.add(req)
-                                session.commit()
-                                log(f"Job with ID {req.id} completed")
-                            except Exception:
-                                status = 'In progress: Light curve not yet available. Waiting for it to complete.'
-                                followup_request.status = status
-                                log(f'Job {req.id}: {status}')
-                                req.last_query = datetime.utcnow()
+                            exitcode = row['exitcode']
+                            exitcode_text = ZTF_PHOTOMETRY_CODES[exitcode]
+
+                            if exitcode in [63, 64, 65, 255]:
+                                req.status = f'No photometry available: {exitcode_text}'
```

Comment:
@mcoughlin is that what we want, or do we want that be the followup request status is set to 'no photometry available', while the facility transaction is set to complete? That way its not fetched anymore when we restart the queue.

Reply:
yeah I had fixed that in the last commit.